cpp/velox/memory/VeloxMemoryPool.cc

```diff
@@ -31,6 +31,22 @@ namespace {
       /* isRetriable */ true,                                       \
       "Exceeded memory manager cap of {} MB",                       \
       (cap) / 1024 / 1024);
+
+std::shared_ptr<velox::memory::MemoryUsageTracker> createMemoryUsageTracker(
+    velox::memory::MemoryPool* parent,
+    velox::memory::MemoryPool::Kind kind,
+    const velox::memory::MemoryPool::Options& options) {
+  if (parent == nullptr) {
+    return options.trackUsage ? velox::memory::MemoryUsageTracker::create(options.capacity, options.checkUsageLeak)
```

Comment:
In Velox, is `options.trackUsage` true by default?

Reply:
Yes, here is [link]( https://github.com/zhejiangxiaomai/velox/blob/main_rebase_417/velox/common/memory/MemoryPool.h#L122)